server/config.json

```diff
@@ -13,11 +13,11 @@
     },
     "json": {
       "strict": false,
-      "limit": "100kb"
+      "limit": "200mb"
```

Comment:
do we still need to increase this limit? Having a large limit can lead to a JSON based DOS attack on the API